indicators/tests/test_views.py

```diff
@@ -0,0 +1,79 @@
+from django.test import TestCase
+from ddt import ddt, data, unpack
+
+from indicators.tests.util import create_user_with_role, USERNAME, PASSWORD
+
+__all__ = ['CountryIndicatorListViewTest', 'CountryIndicatorUpdateTest']
+
+
+_search_data = (
+        ('bra', 1, ['BRA']),
+        ('an', 3, ['ALB', 'FRA', 'GER']),
+        ('ukr', 1, ['UKR']),
+        ('France', 1, ['FRA']),
+        ('Ukraine', 1, ['UKR']),
+        ('tuy', 0, []),
+        ('Frynce', 0, []),
+        (' fj 9', 0, [])
+    )
+
+
+@ddt
+class CountryIndicatorListViewTest(TestCase):
+
+    fixtures = ['indicators/tests_data/test_data.json']
+
+    @classmethod
+    def setUpTestData(cls):
+        role_name = 'Admin'
```

Comment:
Can you remove this magic string or replace it with ADMIN_ID from CultureAnalyzer.py?